vars/Debian.yml

```diff
@@ -1,6 +1,6 @@
 ---
 __redis_package: redis-server
 redis_daemon: redis-server
-redis_conf_dir_mode: 02770
+redis_conf_dir_mode: "02770"
 redis_conf_path: /etc/redis/redis.conf
-redis_conf_mode: 0640
+redis_conf_mode: "0640"
```

Comment:
These changes may actually alter the behavior of the role; I would have to test more but was this just to change some linting output, or because there was a functional problem?